cups/adminutil.c

```diff
@@ -700,6 +702,16 @@ cupsAdminSetServerSettings(
 
   while (cupsFileGetConf(cupsd, line, sizeof(line), &value, &linenum))
   {
+    if (strchr(line, '#') != NULL)
```

Comment:
1. linenum checking is used in order to put blank line in between multilines.
2. I write the if statement on the top so that comments got written on top. and the last else block have again check for # so that that else won't write comments. Since line contains both configuration and comments.